leapp/cli/upgrade/__init__.py

```diff
@@ -198,13 +210,14 @@ def upgrade(args):
 
 
 @command('preupgrade', help='Generate preupgrade report')
-@command_opt('whitelist-experimental', action='append', metavar='ActorName',
-             help='Enables experimental actors')
+@command_opt('whitelist-experimental', action='append', metavar='ActorName', help='Enables experimental actors')
 @command_opt('save-answerfile', help='Path to save custom answerfile')
+@command_opt('debug', is_flag=True, help='Enable debug mode', inherit=False)
+@command_opt('verbose', is_flag=True, help='Enable verbose logging', inherit=False)
 def preupgrade(args):
     if os.getuid():
         raise CommandError('This command has to be run under the root user.')
-
+    parse_verbose(args)
```

Comment:
you're not really parsing it, but handling it

Reply:
also it handles debug too, so this function name is kind of misleading

Reply:
Good points, replaced with `handle_output_level`.